packages/round-manager/src/features/api/services/program.ts

```diff
@@ -0,0 +1,91 @@
+import { ethers } from "ethers"
+import { api } from ".."
+import { global } from "../../../global"
+import { programFactoryContract } from "../contracts"
+import { Program } from "../types"
+
+
+export const programApi = api.injectEndpoints({
+  endpoints: (builder) => ({
+    createProgram: builder.mutation<string, Program>({
+      queryFn: async ({ metadataIdentifier, operatorWallets }) => {
```

Comment:
thoughts on renaming this to `metadataPointer`

Reply:
fixed in #41